python/pyspark/mllib/tree.py

```diff
@@ -383,6 +387,129 @@ def trainRegressor(cls, data, categoricalFeaturesInfo, numTrees, featureSubsetSt
                           featureSubsetStrategy, impurity, maxDepth, maxBins, seed)
 
 
+class GradientBoostedTreesModel(TreeEnsembleModel):
+    """
+    Represents a gradient-boosted tree model.
+
+    EXPERIMENTAL: This is an experimental API.
```

Comment:
Would you mind correcting the experimental tag here and in TreeEnsembleModel and RandomForestModel?  (I know it was there before your PR.)  I'd follow the example in stat.py's chiSqTest:
```
        .. note:: Experimental
```
That makes for better formatting in the generating docs.  Thanks!